packages/frontend-shared/src/store/index.ts

```diff
@@ -0,0 +1,7 @@
+import { createPinia as _createPinia } from 'pinia'
+
+// Reusable installation function, used as an entry point for tests that
+// require an identical setup to main.ts
+export const createPinia = () => {
+  return _createPinia()
+}
```

Comment:
I'm not sure I understand the purpose of this wrapper function. Why not just directly call the library-supplied function from the `beforeEach`?

Reply:
I copied this pattern from our existing pinia testing setup - it's small and it works so I didn't mess with it, but I do see your point 🤔

Reply:
ah, ok, it's just needed for the `export * from './login-connect-store'` to not throw an error,  but that line wasn't there yet. For the sake of consistency, I've added that line to re-export the store like we do in the other package, and have the slightly shorter import paths.